core-bundle/src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -74,7 +74,8 @@ public function output($blnCheckRequest=false)
 	/**
 	 * Return a response object
 	 *
-	 * @param bool $blnCheckRequest If true, check for unsued $_GET parameters
+	 * @param bool $blnCheckRequest 	  If true, check for unsued $_GET parameters
```

Comment:
Is this a typo or was it meant like that ("unsued")? 🙈 

```suggestion
	 * @param bool $blnCheckRequest 	  If true, check for unused $_GET parameters
```

Reply:
See #1114